storm-webapp/src/main/java/org/apache/storm/daemon/logviewer/utils/WorkerLogs.java

```diff
@@ -189,12 +194,29 @@ public String getTopologyOwnerFromMetadataFile(Path metaFile) {
      */
     public Set<String> getAliveIds(int nowSecs) throws IOException {
         return SupervisorUtils.readWorkerHeartbeats(stormConf).entrySet().stream()
-                .filter(entry -> Objects.nonNull(entry.getValue())
-                        && !SupervisorUtils.isWorkerHbTimedOut(nowSecs, entry.getValue(), stormConf))
+                .filter(entry -> Objects.nonNull(entry.getValue()) && !isTimedOut(nowSecs, entry))
```

Comment:
SupervisorUtils .isWorkerHbTimedOut() - is this no longer valid?  Should it be removed or move the timeout code there?

Reply:
SupervisorUtils .isWorkerHbTimedOut() is not referenced. Are you suggesting deleting from SupervisorUtils?

WorkerLogs::isTimedOut gets a cached timeout from WorkerLogs::mapTopologyIdToHeartbeatTimeout cache, so this functionality should probably remain in WorkerLogs.

Reply:
SupervisorUtils .isWorkerHbTimedOut() seems no longer referenced.  If it remains in the code base, we should update to reflect the new topology timeout parameters.  Or else it should be deleted to prevent problems.  Seems like it would be easiest to delete it.